dynamiq/cache/managers/workflow.py

```diff
@@ -3,6 +3,7 @@
 
 from dynamiq.cache.config import CacheConfig
 from dynamiq.cache.managers import CacheManager
+from dynamiq.utils import format_value
 
 
 class WorkflowCacheManager(CacheManager):
```

Comment:
As part of the cache config, is there a way to set TTL?

Reply:
General functionality exists. Should we enable this?

Reply:
Here the runnable config with TTL option
https://github.com/dynamiq-ai/dynamiq/blob/eb7f27bbf197afabb44bca223e9019d2ae3717fe/dynamiq/cache/config.py#L23